Tests: Problems running parts of the test suite

**Describe the bug**

When running the test suite on my workstation with `pytest -vvv -k test_cli_values_geospatial`, it croaks like:

```python
ImportError while loading conftest '/Users/amo/dev/earthobservations/wetterdienst/tests/conftest.py'.
tests/conftest.py:7: in <module>
    from wetterdienst import Settings
wetterdienst/__init__.py:18: in <module>
    from wetterdienst.api import Wetterdienst
wetterdienst/api.py:8: in <module>
    from wetterdienst.provider.dwd.mosmix import DwdMosmixRequest
wetterdienst/provider/dwd/mosmix/__init__.py:4: in <module>
    from wetterdienst.provider.dwd.mosmix.api import DwdMosmixRequest, DwdMosmixValues
wetterdienst/provider/dwd/mosmix/api.py:16: in <module>
    from wetterdienst.core.scalar.values import ScalarValuesCore
wetterdienst/core/scalar/values.py:15: in <module>
    from pint import Quantity
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/pint/__init__.py:14: in <module>
    from .context import Context
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/pint/context.py:15: in <module>
    from .definitions import Definition, UnitDefinition
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/pint/definitions.py:13: in <module>
    from .converters import LogarithmicConverter, OffsetConverter, ScaleConverter
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/pint/converters.py:12: in <module>
    from .compat import HAS_NUMPY, exp, log  # noqa: F401
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/pint/compat.py:163: in <module>
    from xarray import DataArray, Dataset, Variable
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/xarray/__init__.py:1: in <module>
    from . import testing, tutorial
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/xarray/testing.py:9: in <module>
    from xarray.core import duck_array_ops, formatting, utils
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/xarray/core/duck_array_ops.py:28: in <module>
    from .pycompat import cupy_array_type, dask_array_type, is_duck_dask_array
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/xarray/core/pycompat.py:48: in <module>
    dsk = DuckArrayModule("dask")
../../../Library/Caches/pypoetry/virtualenvs/wetterdienst-EkOFQaO8-py3.9/lib/python3.9/site-packages/xarray/core/pycompat.py:24: in __init__
    duck_array_version = Version(duck_array_module.__version__)
E   AttributeError: module 'dask' has no attribute '__version__'
```

Just for the records. When running the full test suite on my workstation with `poe test`, I am receiving this outcome.
```
FAILED tests/test_docs.py::test_readme - assert 1 == 0
FAILED tests/test_api.py::test_api[True-ea-hydrology-kwargs4] - json.decoder.JSONDecodeError: Invalid control character at: line 1 column 528539 (char 528538)
FAILED tests/core/scalar/test_api.py::test_api_dropna - zlib.error: Error -3 while decompressing data: invalid block type
FAILED tests/core/scalar/test_interpolation.py::test_interpolation_temperature_air_mean_200_hourly - AssertionError: DataFrame.iloc[:, 2] (column name="value") are different
```

--- a/tests/provider/dwd/radar/test_api_historic.py
+++ b/tests/provider/dwd/radar/test_api_historic.py
@@ -10,7 +10,6 @@ import h5py
 import pybufrkit
 import pytest
 import requests
-import wradlib as wrl
 
 from wetterdienst.provider.dwd.radar import (
     DwdRadarDataFormat,
@@ -22,6 +21,8 @@ from wetterdienst.provider.dwd.radar import (
 )
 from wetterdienst.provider.dwd.radar.sites import DwdRadarSite
 from wetterdienst.util.datetime import round_minutes
+
+wrl = pytest.importorskip("wradlib", reason="wradlib not installed")
 
 HERE = Path(__file__).parent
 

--- a/tests/provider/dwd/radar/test_api_latest.py
+++ b/tests/provider/dwd/radar/test_api_latest.py
@@ -5,7 +5,6 @@ import re
 from datetime import datetime
 
 import pytest
-import wradlib as wrl
 
 from tests.provider.dwd.radar import station_reference_pattern_unsorted
 from wetterdienst.provider.dwd.radar import DwdRadarValues
@@ -43,6 +42,8 @@ def test_radar_request_composite_latest_rw_reflectivity():
     """
     Example for testing radar COMPOSITES (RADOLAN) latest.
     """
+
+    wrl = pytest.importorskip("wradlib", reason="wradlib not installed")
 
     request = DwdRadarValues(
         parameter=DwdRadarParameter.RW_REFLECTIVITY,
@@ -108,6 +109,8 @@ def test_radar_request_site_latest_dx_reflectivity():
     """
     Example for testing radar SITES latest.
     """
+
+    wrl = pytest.importorskip("wradlib", reason="wradlib not installed")
 
     request = DwdRadarValues(
         parameter=DwdRadarParameter.DX_REFLECTIVITY,

--- a/tests/provider/dwd/radar/test_api_most_recent.py
+++ b/tests/provider/dwd/radar/test_api_most_recent.py
@@ -2,7 +2,6 @@
 # Copyright (C) 2018-2021, earthobservations developers.
 # Distributed under the MIT License. See LICENSE for more info.
 import pytest
-import wradlib as wrl
 
 from wetterdienst.provider.dwd.radar import (
     DwdRadarParameter,
@@ -118,6 +117,8 @@ def test_radar_request_radolan_cdc_most_recent():
     """
     Example for testing radar sites most recent RADOLAN_CDC.
     """
+
+    wrl = pytest.importorskip("wradlib", reason="wradlib not installed")
 
     request = DwdRadarValues(
         parameter=DwdRadarParameter.RADOLAN_CDC,

